src/app.js

```diff
@@ -153,15 +153,8 @@ class App {
     }
     if (deviceName == null) { deviceName = os.hostname() || 'desktop' }
     this.askPassword(function (_, password) {
-      let register = device.registerDeviceSafe
-      register(cozyUrl, deviceName, password, Permissions, function (err, res) {
-        if (err) { callback(err) }
-        let config = {file: true};
-        ({ deviceName } = res);
-        ({ password } = res)
-        let setDesignDoc = filterSDK.setDesignDoc.bind(filterSDK)
-        setDesignDoc(cozyUrl, deviceName, password, config, err => callback(err, res))
-      })
+      // TODO: App.registerRemote() v3
+      callback(null, {deviceName, password})
```

Comment:
It is a good time¹ to use *asynchronism* with `promise` or `await/async`, isn't?

*¹: regarding the fact we use ES2015 and babel*